snuba/datasets/storages/generic_metrics.py

```diff
@@ -119,6 +120,7 @@ def produce_policy_creator() -> DeadLetterQueuePolicy:
         processor=GenericSetsMetricsProcessor(),
         default_topic=Topic.GENERIC_METRICS,
         dead_letter_queue_policy_creator=produce_policy_creator,
+        pre_filter=KafkaHeaderFilter("metric_type", "s"),
```

Comment:
Should we use `InputType.SET.value`?

Reply:
if you think it's clearer, I'm happy to make that change

Reply:
Well it's up to you, I'm not going to insist on it or anything. But isn't that why the enum exists?